RealEstate/apps/core/templates/core/houseEval.html

```diff
@@ -5,96 +5,120 @@
 
 {% block body %}
+{% csrf_token %}
+<div id="heading">
 	<h2><strong> Evaluation for {{house}}</strong> </h2>
 	<h4><em>{{house.address}}</em></h4>
-	<ul class="list-group">
-		{% for category, score in grades %}
-			<li class="list-group-item">
-				<div class="row">
-					<div class="col-sm-6 vcenter">
-						<h4>{{category.summary}}</h4>
-						{% if category.description %}
-							<p>
-								{{ category.description }}
-							</p>
-						{% endif %}
-					</div>
-					<div class="col-sm-6 vcenter rating">
-						<div class="inner-rating" >
-							<center>
-                                <span class="help-text" style="padding-right: 15px">{{min_choice}}</span>
-                                <input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min={{min_score}} data-slider-max={{max_score}} data-slider-step="1" data-slider-value="{% if not score %}{{default_score}}{% else %}{{ score }}{% endif %}"/>
-                                <span class="help-text" style="padding-left: 15px">{{max_choice}}</span>
-							</center>
-						</div>
+</div>
+<p id="save-status" class="pull-right"></p>
+<ul class="list-group">
+	{% for category, score in grades %}
+		<li class="list-group-item">
+			<div class="row">
+				<div class="col-sm-6 vcenter">
+					<h4>{{category.summary}}
+						<span id='update_{{category.id}}' class='glyphicon glyphicon-ok icon-ok'></span>
+					</h4>
+					{% if category.description %}
+						<p>
+							{{ category.description }}
+						</p>
+					{% endif %}
+				</div>
+				<div class="col-sm-6 vcenter rating">
+					<div class="inner-rating" >
+						<center>
+							<span class="help-text" style="padding-right: 15px">{{min_choice}}</span>
+							<input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min={{min_score}} data-slider-max={{max_score}} data-slider-step="1" data-slider-value="{% if not score %}{{default_score}}{% else %}{{ score }}{% endif %}"/>
+							<span class="help-text" style="padding-left: 15px">{{max_choice}}</span>
+						</center>
 					</div>
 				</div>
-			</li>
-		{% endfor %}
-	</ul>
-	<form style="margin-bottom:20px" action="" role="form" method="post">
-		{% bootstrap_form form%}
-		{% csrf_token %}
-		<div style="margin-top: 40px">
-			<center>
-				{% buttons submit='Save Evaluation' %}{% endbuttons %}
-				{% bootstrap_messages %}
-
-			</center>
-		</div>
-	</form>
+			</div>
+		</li>
+	{% endfor %}
+</ul>
 {% endblock %}
 
 {% block scripts %}
 <script src="{% static "slide.js" %}"></script>
 <script>
-    var jsScores = {{js_scores|safe}};
-	$(".slide").slider({
-		formatter: function(value) {
-            return jsScores[value] || "";
+var jsScores = {{js_scores|safe}}
+$(".slide").slider({
+	formatter: function(value) {
+          return jsScores[value] || "";
+	}
+});
+$(".slide").on("change", function(slideEvt) {
+	var timer = setTimeout(showStatus, 1500);
+	clearTimeout(timer);
```

Comment:
The timeout is set and then instantly cleared...?  Can be replaced by:
```javascript
var timer;
```
and deleting the clearTimeout.